app/notification_test.go

```diff
@@ -2821,3 +2822,74 @@ func TestReplyPostNotificationsWithCRT(t *testing.T) {
 		assert.Nil(t, membership)
 	})
 }
+
+func TestChannelAutoFollowThreads(t *testing.T) {
+	th := Setup(t).InitBasic()
+	defer th.TearDown()
+
+	u1 := th.BasicUser
+	u2 := th.BasicUser2
+	u3 := th.CreateUser()
+	th.LinkUserToTeam(u3, th.BasicTeam)
+	c1 := th.BasicChannel
+	th.AddUserToChannel(u2, c1)
+	th.AddUserToChannel(u3, c1)
+
+	// Set auto-follow for user 2
+	member, appErr := th.App.UpdateChannelMemberNotifyProps(th.Context, map[string]string{model.ChannelAutoFollowThreads: "true"}, c1.Id, u2.Id)
+	require.Nil(t, appErr)
+	require.Equal(t, "true", member.NotifyProps[model.ChannelAutoFollowThreads])
+
+	// Enable CRT
+	os.Setenv("MM_FEATUREFLAGS_COLLAPSEDTHREADS", "true")
+	defer os.Unsetenv("MM_FEATUREFLAGS_COLLAPSEDTHREADS")
```

Comment:
I don't think this is necessary any more.

Reply:
Done.